plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -2006,7 +2251,8 @@ func TestEBSVolumeCountConflicts(t *testing.T) {
 	expectedFailureReasons := []algorithm.PredicateFailureReason{ErrMaxVolumeCountExceeded}
 
 	for _, test := range tests {
-		pred := NewMaxPDVolumeCountPredicate(filter, test.maxVols, pvInfo, pvcInfo)
+		os.Setenv(KubeMaxPDVols, strconv.Itoa(test.maxVols))
```

Comment:
Do you need to restore the previous value like in the original test?

Reply:
fixed, thanks!